doc/authoring_command_modules/authoring_commands.md

```diff
@@ -1,5 +1,10 @@
 The document provides instructions and guidelines on how to author individual commands.
 
+> **Note**
```

Comment:
Is it better to list the benefits here as well as below?
```
[new codegen tool](https://azure.github.io/aaz-dev-tools/) is UI based and easy to use. We encourage your team to use it to generate CLI commands automatically as nobody knows your logic better than your team. Using our new codegen tool, you can have the following benefits:


- Shorten development cycle (save 50%+ development time, if no block issues, usually it takes less than 1 week).
- Directly generate CLI commands from swagger instead of waiting for SDK release.
- Consistent code style improves command quality and avoids code review efforts.
```